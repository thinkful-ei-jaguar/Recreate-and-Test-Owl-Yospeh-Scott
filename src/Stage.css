
.participantsStage {
  position: absolute;
  top: 0;
  left: 25vw;
  width: 100%;
  background-color: slategray;
}