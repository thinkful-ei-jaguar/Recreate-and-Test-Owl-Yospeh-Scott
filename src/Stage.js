import React from 'react';
import './Stage.css';

export default function Stage(props) {
  return (
    <div className="participantsStage">
      <h3>{props.name}</h3>
      <h3><img src={props.avatar} alt='avatar'></img></h3>
    </div>
  )
}