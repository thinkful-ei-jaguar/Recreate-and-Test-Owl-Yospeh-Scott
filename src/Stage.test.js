import React from 'react';
import ReactDOM from 'react-dom';
import Stage from './Stage'
import renderer from 'react-test-renderer';


describe('Stage Component', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(
    <Stage 
        cards={[
          {id: 1, name: 'Yosef', avatar: 'https://robohash.org/itaquevoluptatumrerum.jpg?size=200x200&set=set1'},
          {id: 2, name: 'Scott', avatar: 'https://robohash.org/doloredolorescupiditate.jpg?size=200x200&set=set1' }
         ]}
      />, 
      div
    );
    ReactDOM.unmountComponentAtNode(div);
  });

  it('renders the UI as expected', () => {
    const tree = renderer
      .create(
      <Stage 
        header='List 1'
        cards={[
          {id: 1, name: 'Yosef', avatar: 'https://robohash.org/itaquevoluptatumrerum.jpg?size=200x200&set=set1'},
          {id: 2, name: 'Scott', avatar: 'https://robohash.org/doloredolorescupiditate.jpg?size=200x200&set=set1'}
         ]}
      />
      )
      .toJSON();
    expect(tree).toMatchSnapshot();  
  });
});