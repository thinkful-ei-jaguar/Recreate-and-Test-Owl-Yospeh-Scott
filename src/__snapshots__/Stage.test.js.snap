// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Stage Component renders the UI as expected 1`] = `
<div
  className="participantsStage"
>
  <h3 />
  <h3>
    <img
      alt="avatar"
    />
  </h3>
</div>
`;
